packages/cef/data_stream/log/elasticsearch/ingest_pipeline/cp-pipeline.yml

```
@@ -285,6 +285,23 @@ processors:
   - remove:
       field: _tmp_copy
 
+  - set:
+      field: email.to.address
+      value: ["{{email.to.address}}"]
```

```suggestion
      value: ["{{{email.to.address}}}"]
```

Use triple braces to avoid unexpected HTML escaping. https://mustache.github.io/mustache.5.html